src/main/java/org/folio/orders/utils/ErrorCodes.java

```diff
@@ -32,6 +32,7 @@
   ORDER_VENDOR_IS_INACTIVE("vendorIsInactive", "Order cannot be open as the associated vendor is inactive"),
   POL_ACCESS_PROVIDER_IS_INACTIVE("accessProviderIsInactive", "Order cannot be open as the associated access provider is inactive"),
   ORDER_VENDOR_NOT_FOUND("vendorNotFound", "Order cannot be open as the associated vendor not found"),
+  VENDOR_ISSUE("vendorIssue", "Order cannot be updated due to issues related to associated vendor/access provider"),
```

Comment:
All we are using this code is when the we are unable to verify status, I think we can be more specific with the message.

Reply:
see above comments.  If needed we can address in a follow-up story